packages/@aws-cdk/aws-ecs/lib/log-drivers/splunk-log-driver.ts

```diff
@@ -0,0 +1,183 @@
+import { Construct } from '@aws-cdk/core';
+import { ContainerDefinition } from '../container-definition';
+import { LogDriver, LogDriverConfig } from "./log-driver";
+
+/**
+ * Specifies the splunk log driver configuration options.
+ */
+export interface SplunkLogDriverProps {
+  /**
+   * Splunk HTTP Event Collector token.
+   */
+  readonly token: string;
+
+  /**
+   * Path to your Splunk Enterprise, self-service Splunk Cloud instance, or Splunk
+   * Cloud managed cluster (including port and scheme used by HTTP Event Collector)
+   * in one of the following formats: https://your_splunk_instance:8088 or
+   * https://input-prd-p-XXXXXXX.cloud.splunk.com:8088 or https://http-inputs-XXXXXXXX.splunkcloud.com.
+   */
+  readonly url: string;
+
+  /**
+   * Event source.
+   *
+   * @default - source not set.
+   */
+  readonly source?: string;
+
+  /**
+   * Event source type.
+   *
+   * @default - sourceType not set.
+   */
+  readonly sourceType?: string;
+
+  /**
+   * Event index.
+   *
+   * @default - index not set.
+   */
+  readonly index?: string;
+
+  /**
+   * Path to root certificate.
+   *
+   * @default - capath not set.
+   */
+  readonly capath?: string;
+
+  /**
+   * Name to use for validating server certificate; by default the hostname of the splunk-url is used.
+   *
+   * @default - caname not set.
+   */
+  readonly caname?: string;
+
+  /**
+   * Ignore server certificate validation.
+   *
+   * @default - insecureskipverify not set.
+   */
+  readonly insecureskipverify?: string;
```

Comment:
`insecureSkipVerify`. But why not `tlsSkipVerify` like the other class? I think that's a better name.

Reply:
Only reason is that’s the name that’s given in the documentation. I figured I’d err closer to that, but I agree the other is a batter name.

Reply:
Yeah, I guess we stick to the doc name then. But please make it `camelCase` anyway.